This is a temporary patch for:

Bug 38084012 - ps2pdf(1) no longer creates valid pdf files

I have filed this upstream issue:

Bug 708620 - ps2pdf generates damaged PDF
https://bugs.ghostscript.com/show_bug.cgi?id=708620

--- ghostscript-10.05.1/base/stdint_.h
+++ ghostscript-10.05.1/base/stdint_.h
@@ -150,7 +150,7 @@
 #endif
 
 #if defined(HAVE_INTTYPES_H) && HAVE_INTTYPES_H == 1
-# include <inttypes.h>
+# define HAVE_INTTYPES_H 1
 #else
 # if defined(__WIN32__)
 #  define PRId32 "I32d"
